### PR TITLE
PAS 1092 : Build OSP XRS work thread build

### PR DESCRIPTION
**Pull request by shinealex33**
PAS-1092

New feature

Build OSP XRS work thread to build XRS exchange rate call from currency conversion service.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -142,6 +145,19 @@ auditing {
     }
 }
 
+workers {
+  xrs-exchange-rate {
+
+    initial-delay = 0.seconds
+    interval      = 1.minutes
+    parallelism   = 0
```
Comment (awasnik) [inline conf/application.conf:153]: please use this value and set it to 1 and remove unused code

File: test/uk/gov/hmrc/currencyconversion/workers/XrsExchangeRateRequestWorkerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    )
+    val app = builder.build()
+    running(app) {
+      val worker = app.injector.instanceOf[XrsExchangeRateRequestWorker]
+
+      val workerResponse = worker.tap.pull.futureValue.value
+      workerResponse.status mustBe SERVICE_UNAVAILABLE
+    }
+  }
+}
```
Comment (awasnik) [inline test/uk/gov/hmrc/currencyconversion/workers/XrsExchangeRateRequestWorkerSpec.scala:89]: end of line please

**Review by awasnik — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (shinealex33) [reply conf/application.conf]: done

File: test/uk/gov/hmrc/currencyconversion/workers/XrsExchangeRateRequestWorkerSpec.scala (hunk 2 above)
Comment (shinealex33) [reply test/uk/gov/hmrc/currencyconversion/workers/XrsExchangeRateRequestWorkerSpec.scala]: Done. thanks!

**Pull request merged**